test/integration/supervisor_login_test.go

```diff
@@ -176,7 +176,7 @@ func TestSupervisorLogin(t *testing.T) {
 			},
 			// the ID token Subject should be the Host URL plus the value pulled from the requested UserSearch.Attributes.UID attribute
 			wantDownstreamIDTokenSubjectToMatch: regexp.QuoteMeta(
-				"ldaps://" + env.SupervisorUpstreamLDAP.Host + "?sub=" + env.SupervisorUpstreamLDAP.TestUserUniqueIDAttributeValue,
+				"ldaps://" + env.SupervisorUpstreamLDAP.StartTLSOnlyHost + "?sub=" + env.SupervisorUpstreamLDAP.TestUserUniqueIDAttributeValue,
```

Comment:
I noticed this is somewhat weird for Jumpcloud, since all users have the same `ldaps://ldap.jumpcloud.com/?sub=XYZ`-style URL, and any `sub` collisions would be the same across different search bases. Maybe we should encode the search base somehow?

Reply:
Issue #629 opened.